samples/nrf9160/modem_shell/src/gnss/gnss_shell.c

```diff
@@ -1031,11 +1059,15 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	sub_gnss_agps,
 	SHELL_CMD(automatic, &sub_gnss_agps_automatic,
 		  "Enable/disable automatic fetching of AGPS data.", cmd_gnss_agps_automatic),
-	SHELL_CMD_ARG(inject, NULL, "Fetch and inject AGPS data to GNSS.",
-		      cmd_gnss_agps_inject, 1, 0),
+	SHELL_CMD_ARG(inject, NULL, "<assistance_data>\nFetch and inject AGPS data to GNSS. "
```

Comment:
At least I have tried to keep lines short enough so that lines would not be wrapped in the terminal. This line is more than 200 characters long, so maybe it should be split on several lines?

Reply:
And what was the convention for different arguments in mosh, was it so that `<>`  denotes a mandatory argument, while `[]` is an optional argument?

Reply:
I don't remember if we made a decision for `<>` and `[]` but that sounds reasonable :)

Reply:
SHELL_CMD_ARG seems to handle the line wrapping nicely, no need to start adding \n here imo. I will change <> to []

Reply:
Yep, wrapping works fine as long as resize has been run so I think it's fine if we don't split these long help comments into separate lines.